AppCenterData/AppCenterDataTests/MSDBDocumentStoreTests.m

```diff
@@ -407,8 +415,10 @@ - (void)testDeleteExistingAppDocument {
 - (void)testDeleteExistingUserDocument {
 
   // If
-  MSDocumentWrapper *documentWrapper = [MSDocumentUtils documentWrapperFromData:[self jsonFixture:@"validTestDocument"]
+  MSDocumentWrapper *documentWrapper = [MSDocumentUtils documentWrapperFromData:[self jsonFixture:@"validTestUserDocument"]
                                                                    documentType:[MSDictionaryDocument class]
+                                                                      partition:@"user-123"
```

Comment:
You don't need a custom prefixed user partition it's only used for the user table in the storage. These calls are more likely to use `kMSDataUserDocumentsPartition`.

Reply:
I was wrong the returned metadata from the document wrapper should contain the user-account.